src/MLOps.NET/Storage/Deployments/SchemaGenerator.cs

```diff
@@ -0,0 +1,36 @@
+﻿using ICSharpCode.Decompiler;
+using ICSharpCode.Decompiler.CSharp;
+using ICSharpCode.Decompiler.TypeSystem;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace MLOps.NET.Storage.Deployments
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public sealed class SchemaGenerator
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="className"></param>
+        /// <returns></returns>
+        public string GenerateDefinition(string className)
```

Comment:
This is great @AnoojNair!

A couple of things:
- Can we pass in the type as a generic instead, e.g. `GenerateDefinition<T>(string className)` 
The reason I want to do that is className will be the name of the new class, but we need to fetch the name of class to base everything off from the type argument (they are going to be different names)

- Could we check the entire AppDomain instead of just the calling assembly? I think the calling assembly may just be `MLOps.NET` which is not going to contain this class, but if we look through all assemlies in the appdomain we should find it

Reply:
@aslotte , Thanks for the feedback, I will make the changes.